test/functional/plugin/lsp/inline_completion_spec.lua

```diff
@@ -0,0 +1,231 @@
+local t = require('test.testutil')
+local n = require('test.functional.testnvim')()
+local t_lsp = require('test.functional.plugin.lsp.testutil')
+local Screen = require('test.functional.ui.screen')
+
+local dedent = t.dedent
+
+local api = n.api
+local exec_lua = n.exec_lua
+local insert = n.insert
+local feed = n.feed
+
+local clear_notrace = t_lsp.clear_notrace
+local create_server_definition = t_lsp.create_server_definition
+
+describe('vim.lsp.inline_completion', function()
+  local text = dedent([[
+function fibonacci()
+]])
```

Comment:
dedent() allows indenting the values to be more readable
```suggestion
  local text = dedent([[
    function fibonacci()
    ]])
```